.cspell.json

```diff
@@ -0,0 +1,113 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
+    "ciphertext",
+    "evenodd",
+    "gitea",
+    "gonic",
+    "Netrc",
+    "prismjs",
+    "rawurl",
+    "Refspec",
+    "securecookie",
+    "sess",
+    "techknowlogick",
+    "vueuse",
+    "windicss",
+    "xorm",
+    "xormigrate",
+    "zerolog",
+    "Msgf",
+    "varchar",
+    "autoincr",
+    "stretchr",
+    "bradrydzewski",
+    "octocat",
+    "brightbox",
+    "anbraten",
+    "Codeberg",
+    "Fediverse",
+    "Weblate",
+    "Hetzner",
+    "windi",
+    "intlify",
+    "unplugin",
+    "Pinia",
+    "woodpeckerci",
+    "tinycolor",
+    "favicons",
+    "Laszlo",
+    "Fogas",
+    "Georgiana",
+    "Ionescu",
+    "typecheck",
+    "HTTPFS",
+    "Println",
+    "ppid",
+    "xlog",
+    "Debugf",
+    "Warnf",
+    "Infof",
+    "Wrapf",
+    "urfave",
+    "creativecommons",
+    "compatiblelicenses",
+    "sublicensable",
+    "waivable",
+    "kyvg",
+    "iconify",
+    "Upsert",
+    "Rydzewski",
+    "fsnotify",
+    "tink",
+    "desaturate",
+    "golangci",
+    "Kaniko",
+    "Reviewdog",
+    "DATASOURCE",
+    "httpsig",
+    "envsubst",
+    "TARGETOS",
+    "TARGETARCH",
+    "GOARCH",
+    "Curr",
+    "doublestar",
+    "multierr",
+    "markdownlint",
+    "buildx",
+    "HEALTHCHECK",
+    "devx",
+    "gomod",
+    "laszlocph"
+  ],
+  "ignorePaths": [
+    "**/node_modules/**/*",
+    "pnpm-lock.yaml",
+    ".gitignore",
+    ".git/**/*",
+    ".cspell.json",
+    ".golangci.yaml",
+    ".vscode/extensions.json",
+    "web/src/assets/locales/**/*",
+    "web/components.d.ts",
+    "server/store/datastore/migration/**/*",
+    "go.mod",
+    "go.sum",
+    "*.excalidraw",
+    "CHANGELOG.md",
```

Comment:
Should probably be also be checked in the future. It will fail in the release PR so we will know if we need to rename a PR.

Reply:
Checking commit messages should probably be done before merging PRs then. Will move it down to the TODOs